scripts/whitelist-fiscal-code-script/src/insert-fiscal-codes/insert-fiscal-codes.ts

```diff
@@ -0,0 +1,125 @@
+import { CosmosClient } from '@azure/cosmos';
+import * as CsvStringify from 'csv-stringify';
+import fs from 'fs';
+import { getArgvParam } from '../utils/get-argv-param';
+
+const writeUpsertedFiscalCodes = async (
+  upsertedFiscalcodes: string[],
+): Promise<void> => {
+  const data = [['fiscalCode'], ...upsertedFiscalcodes.map((item) => [item])];
+
+  const options = {
+    header: false,
+    quoted: false,
+  };
+
+  CsvStringify.stringify(data, options, (error, output) => {
+    if (!error) {
+      fs.writeFileSync(
+        `${getArgvParam('--outputDir') ?? 'logs'}/whitelisted_fiscal_codes.csv`,
+        output,
+      );
+      console.log(
+        'whitelisted fiscal codes written to logs/whitelisted_fiscal_codes.csv',
```

Comment:
I would make the message dynamic. If I choose a different outputDir, the message here remains the same